pkg/resizer/csi_resizer.go

```diff
@@ -296,11 +294,12 @@ func timeoutCtx(timeout time.Duration) (context.Context, context.CancelFunc) {
 }
 
 func getCredentials(k8sClient kubernetes.Interface, ref *v1.SecretReference) (map[string]string, error) {
+	var ctx context.Context = context.Background()
```

Comment:
We should probably use some configurable timeout for all these contexts

Reply:
sure. as discussed I will start with context.TODO(), then a follow up PR to configurable timeout .. thanks :+1:

Reply:
Golang nit - this can be written much shorter as:
```
ctx := context.TODO()
```

Reply:
@pohly sure, I am updating all the function signatures to use context.TODO() directly  ( For examples `*.Get(context.TODO(), ....)` )as discussed. On it now. One by one I will update the PR. Thanks!